e2e/e2etests/helpers.go

```diff
@@ -90,6 +65,42 @@ func withdrawBTCZRC20(r *runner.E2ERunner, to btcutil.Address, amount *big.Int)
 	return rawTx
 }
 
+// approveAndWithdrawBTCZRC20 is a helper function to call 'approve' and 'withdraw' on BTCZRC20 contract
+func approveAndWithdrawBTCZRC20(r *runner.E2ERunner, to btcutil.Address, amount *big.Int) *ethtypes.Receipt {
```

Comment:
Let's make an approval optional

```suggestion
func (r *Runner) BTCWithdraw(to btcutil.Address, amount *big.Int, approve bool) *ethtypes.Receipt {
```